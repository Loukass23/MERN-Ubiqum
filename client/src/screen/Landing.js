import React from 'react'
import logo from '../assets/MYtineraryLogo.png'

const Landing = () => {
    return (
        <div>
            <img src={logo} width={300} alt="" />
            <h1>Wassup</h1>
        </div>
    )
}

export default Landing
