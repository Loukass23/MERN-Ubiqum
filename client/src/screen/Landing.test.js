import React from 'react';
import { shallow, render } from 'enzyme';
import Landing from './Landing';
import toJson from 'enzyme-to-json';

it('Landing component should render without crashing', () => {
    shallow(<Landing />);
});

it("landing component snapshot shallow ", () => {
    const wrapper = shallow(<Landing />);
    expect(wrapper).toMatchSnapshot();
});

it("landing component snapshot render", () => {
    const wrapper = render(<Landing />);
    expect(wrapper).toMatchSnapshot();
});
