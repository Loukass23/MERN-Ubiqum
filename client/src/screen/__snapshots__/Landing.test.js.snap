// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landing component snapshot render 1`] = `
<div>
  <img
    alt=""
    src="MYtineraryLogo.png"
    width="300"
  />
  <h1>
    MyItinerary
  </h1>
  <p>
    Find your perfect trip, designed by insiders who know and love their cities.
  </p>
  <section>
    <h2>
      Start Browsing
    </h2>
    <img
      alt="circle arrow"
      class="circledArrow"
      src="circled-right-2.png"
      width="50"
    />
  </section>
  <section>
    <p>
      Want to build your own MYtinerary?
    </p>
    <div
      class="Links"
    >
      <a
        class="Home-link"
        href=""
        rel="noopener noreferrer"
        target="_blank"
      >
        Create Account
      </a>
    </div>
  </section>
</div>
`;

exports[`landing component snapshot shallow  1`] = `
<div>
  <img
    alt=""
    src="MYtineraryLogo.png"
    width={300}
  />
  <h1>
    MyItinerary
  </h1>
  <p>
    Find your perfect trip, designed by insiders who know and love their cities.
  </p>
  <section>
    <h2>
      Start Browsing
    </h2>
    <img
      alt="circle arrow"
      className="circledArrow"
      src="circled-right-2.png"
      width={50}
    />
  </section>
  <section>
    <p>
      Want to build your own MYtinerary?
    </p>
    <div
      className="Links"
    >
      <a
        className="Home-link"
        href=""
        rel="noopener noreferrer"
        target="_blank"
      >
        Create Account
      </a>
    </div>
  </section>
</div>
`;
